i18n/zh-cn/docusaurus-plugin-content-docs/version-v2.0/ops/version-maintain-plan.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 版本维护计划
+keywords: [Seata,维护,多版本]
+description: Seata版本维护计划.
+---
+
+# Apache Seata(incubating)版本维护计划
+
+------
+
+
+![version maintain plan](/img/version/version_maintain_plan.png)
+
+（上图为Apache Seata(incubating)历史版本维护的状态图）
+
+需要注意的是，社区仅维护当前时间点下最新的3个版本，并且对于非当前版本，只提供安全漏洞和bugfix相关的迭代，鼓励用户尝试最新版本的feature并给出反馈。
```

Comment:
How is the period calculated on the picture here, and why does 1.7 have a 6 month maintenance period and 1.8 only 3 months?

Reply:
> How is the period calculated on the picture here, and why does 1.7 have a 6 month maintenance period and 1.8 only 3 months?

This is calculated through the time node of the community release. Our maintenance time is until the release of the next official version, so the span of 1.7 is very large.

Reply:
Based on the [release dates](https://github.com/apache/incubator-seata/releases), considering that version 1.6.0 was released in December, and version 1.7.0 was released in the following July, the period between these releases is considered the stability maintenance period for version 1.6.0. With version 1.8.0 being released in October, the stability maintenance period for version 1.7.0 is between July and October.